data/persons.txt

```diff
@@ -1850,3 +1849,164 @@ ship "Modified Ladybug"
+			"Now there's a lot less detail conflicting with one another."
+
+#		tibetiroka
+			"It's just the Steam CI, no need to panic."
+			"Did you check for nullptr?"
+			"		How do we indent after line breaks again?"
+			"Please put the includes in alphabetical order."
+			"Always keep three empty lines between functions."
+			"Why do Aliens speak English?"
+
+#		1010todd
+			"Style check."
+			"I'll give this a better look later."
+			"What is this for?"
+			"This should be a private member."
+			"Comments?"
+			"Please remove unnecessary space."
+
+			"Looks good to me."
+			"Is this the best name?"
+			"The standard copyright header was changed recently."
+
+			"Please use the template."
+			"The decision to implement fancy quote is one of the worst in computer history."
+			"	This needs another indent."
+#		EjoThims
+	phrase
+		word
+			"If you want your pull request to not be merged"
+			"If you prefer your pull request never be merged"
+			"If you prefer your pull request sits idle"
+			"If you want your pull request to be closed"
+		word
+			", "		
+		word
+			"make sure you keep adding content to it"
+			"always change unrelated things"
+			"do not ever test it and don't add integration tests. Whatever those even are, they're apparently important"
+			"try your best to avoid respecting existing syntax and conventions"
+			"just forget about it for a while"
+			"use the forbidden word: 'realism'"
+			"bury the new feature in semi-relevant data changes"
+			"do not, under any circumstances, ever be willing to compromise"
+			"make sure to completely ignore what the reviewers tell you to do"
+			"skip over writing an issue about it first"
+		word
+			"!"
+			"."
+			". Trust me! It works."
+			". You shouldn't do those things, though! I'm just kidding."
+
+	phrase
+		word
+			"Just a syntax review. Hold on, I gotta rewrite this quickly. Can you delay the merge?"
+			"People who complain about HR don't actually appreciate what we folks do. The number of pull requests that went into that baby..."
+			"I was requested for spelling? Oh my goodness, this needs a rewrite..."
+			"One more review. Pretty please?"
+			"Just one more review before merging."
+			"Capitalise that proper noun. Use American English for consistency. For goodness' sake, that needs a comma!"
+			"Comma here. Comma doesn't go there. I think that one needs a comma? Hold on, what was a comma again?"
+			"Unidentified sentence appears to be lacking comma. Very dangerous, highly likely to incite multiple suggestions on where comma should go. Calling backup reviewers."
+			"Commas are your greatest friends."
+			"Commas are your greatest enemies."
+
+	ship "Waverider" "Pain of the Proofreader"
+		"never disabled"
+	ship "Waverider" "Regret of the Art Critics"
+		"never disabled"
+	ship "Waverider" "Curse of the Code Herders"
+		"never disabled"
+	ship "Waverider" "Tears of the Choreographers"
+		"never disabled"
+	ship "Waverider" "Blood of the Adjuncators"
```

Comment:
What is an Adjuncator?

Reply:
> An adjudicator is someone who presides, [judges](https://en.wikipedia.org/wiki/Judge), and [arbitrates](https://en.wikipedia.org/wiki/Arbitration) during a formal [dispute](https://en.wiktionary.org/wiki/dispute) or [competition](https://en.wikipedia.org/wiki/Competition). They have numerous purposes, including preliminary [legal judgments](https://en.wikipedia.org/wiki/Legal_judgment), to determine applicant eligibility, or to assess contenders' performance in competitions.

For this purpose, they represent either themes or mission.

Reply:
Alright.
```suggestion
	ship "Waverider" "Blood of the Adjudicators"
```